lib/DraggableCore.es6

```diff
@@ -319,18 +320,31 @@ export default class DraggableCore extends React.Component {
     return this.handleDragStop(e);
   };
 
+  removeScroll: EventHandler<MouseEvent> =  (e) => {
+    e.preventDefault();
+    return false;
+  };
+
   // Same as onMouseDown (start drag), but now consider this a touch device.
   onTouchStart: EventHandler<MouseEvent> = (e) => {
     // We're on a touch device now, so change the event handlers
     dragEventFor = eventsFor.touch;
 
+    // Stop the touch device from scrolling while user is dragging.
+    if (this.state.dragging) {
```

Comment:
I don't believe this will ever fire. `this.state.dragging` is set to true in `handleDragStart`.